src/databricks/labs/lsql/dashboards.py

```diff
@@ -377,6 +410,9 @@ def _get_layouts(widgets_metadata: list[WidgetMetadata]) -> list[Layout]:
             placed_tile = tile.place_after(position)
             layout = Layout(widget=placed_tile.widget, position=placed_tile.position)
```

Comment:
@nfx : Maybe we want to convert the `widget` on a tile to `widgets`. Might be useful in future cases, or at least for multiple filters (on different columns)